src/components/gridlines.ts

```diff
@@ -67,10 +146,25 @@ namespace Plottable.Components {
       return this;
     }
 
+    private _mkTicks(scale: QuantitativeScale<any> | Plottable.Scales.Category): () => ScaleDomainValues {
```

Comment:
i think the original author intended it to mean "make". I'll change to `_makeTicks`